docs/reference/cli-commands/help.md

```diff
@@ -8,18 +8,47 @@ information about available `pebble` commands.
 
 To display a summary about Pebble and the available commands, run:
 
-```bash
-pebble -h
+```{terminal}
+:input: pebble -h
+Pebble lets you control services and perform management actions on
+the system that is running them.
+
+Usage: pebble <command> [<options>...]
+
+Commands can be classified as follows:
+
+         Run: run
+        Info: help, version
+        Plan: add, plan
+    Services: services, logs, start, restart, signal, stop, replan
+      Checks: checks, health
+       Files: push, pull, ls, mkdir, rm, exec
+     Changes: changes, tasks
+     Notices: warnings, okay, notices, notice, notify
+  Identities: identities --help
```

Comment:
Can `help.md` be generated with automatic code blocks too? Otherwise this will have to be updated whenever we add a command, which we might forget. We might need a special case in the script, as you need to type `pebble help` or `pebble --help` to get it, rather than the usual pattern of `pebble help --help`.

Reply:
Yes, that's why I left it out. I'll add some logic in the script to unify it.